conftest.py

```diff
@@ -41,14 +47,20 @@ def get_task_ids_for_user(user: str):
             yield task[4]
 
 
-def get_task_logs_for_id(task_id: str, lines: int=1000000):
+def get_task_logs_for_id(task_id: str) -> str:
+    """ Thin wrapper to return the logs for a given task.
+    Note: 10k lines is used as this will typically be 1-3MB. Higher volume
+    log pulls could result in reporting difficulties for other tests or CI
+    """
     return subprocess.check_output([
-        'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
+        'dcos', 'task', 'log', task_id, '--lines', '10000']).decode()
```

Comment:
I'd actually like to leave this really high given the behavior of `dcos task log` is just to grab the last stdout file. WDYT?